### ISSUE TITLE
Wrongly placed semicolon when writing nested objects

### ISSUE THREAD

**Issue by netei** [eventualbuddha/decaffeinate#91]
Similar to #90 

Input: 

```
f=
	foo:
		bar:"baz"
```

Output: 

```
var f=
	{foo:
		{bar:"baz"};
}
```

Expected: 

```
var f=
	{foo:
		{bar:"baz"}
};
```

**Comment by eventualbuddha** [comment]
I am seeing a different [result](http://eventualbuddha.github.io/decaffeinate/repl/#?evaluate=true&code=f%3D%0A%20%20%20%20foo%3A%0A%20%20%20%20%20%20%20%20bar%3A%22baz%22) than you:

```js
var f=
    {foo:
        {bar:"baz"}};
```

This output isn't great stylistically, but it's correct syntactically. What version are you using? The [repl](http://eventualbuddha.github.io/decaffeinate/repl) is currently at v1.40.18.

**Comment by netei** [comment]
It's because I'm testing it in a file and have a new line . after my statement

I can reproduce with this : 

```
f=
  foo:
    ar:"baz"
  
c=2
```

which outputs this:

```
var f=
  {foo:
    {ar:"baz"};
  
}var c=2;
```

Link : http://eventualbuddha.github.io/decaffeinate/repl/#?evaluate=true&code=f%3D%0A%20%20foo%3A%0A%20%20%20%20ar%3A%22baz%22%0A%20%20%0Ac%3D2

**Comment by eventualbuddha** [comment]
Ah, yep. Thanks!

**Comment by netei** [comment]
I think you forgot to publish the new version on npm : https://www.npmjs.com/package/decaffeinate is still 1.40.18

**Comment by eventualbuddha** [comment]
@netei You're right. Publishing soon…

**Comment by eventualbuddha** [comment]
Fixed in v1.41.1.
